freqtrade/freqai/data_kitchen.py

```diff
@@ -68,14 +69,33 @@ def __init__(self, config: Dict[str, Any], dataframe: DataFrame, live: bool = Fa
                 config["freqai"]["backtest_period"],
             )
 
+    def assert_config(self, config: Dict[str, Any], live: bool) -> None:
+        assert config.get('freqai'), "No Freqai parameters found in config file."
```

Comment:
Please don't use "assert" in non-test code (basically outside of the `tests/` directory.

It can be disabled via Python flag (`python -O`) - converting all of these asserts into "noop" statements.

Reply:
Ok thanks. What method would you prefer that I use instead to verify the config has the necessary information?

Reply:
I guess I can just test for it and then log an error if it doesnt pass?

Reply:
I'm not certain you'll need explicit asserts for this (at least not for all options - maybe only for freqai (the parent component) - as you can't require that without breaking all existing configs).

a properly configured json schema (`constants.py` - `'required'`) usually fills this need already (check if things are there, correct type, within a certain range, ...) - _unless_ it's some complex dependencies (if A is set, B can only be X or Y, otherwise it can be Z too) ...

Reply:
Moved validation to `config_validation.py`. Should be resolved.